revisions/announcer/announcer.go

```diff
@@ -0,0 +1,293 @@
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package announcer
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"log"
+
+	"github.com/web-platform-tests/wpt.fyi/revisions/epoch"
+	agit "github.com/web-platform-tests/wpt.fyi/revisions/git"
+	git "gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/config"
+	"gopkg.in/src-d/go-git.v4/plumbing"
+	"gopkg.in/src-d/go-git.v4/plumbing/object"
+	"gopkg.in/src-d/go-git.v4/plumbing/storer"
+	"gopkg.in/src-d/go-git.v4/storage/memory"
+)
+
+const mergedPrTagPrefix = "refs/tags/merge_pr_"
+
+var errNotAllEpochsConsumed = errors.New("Not all epochs consumed")
+var errNilRepo = errors.New("Repository may not be nil")
+var errVacuousEpochs = errors.New("[]epoch.Epoch slice is vacuous: contains no epochs")
```

Comment:
Nit: Simpler naming is better (`empty` vs `vacuous`)

Reply:
Done.